src/fromager/resolver.py

```diff
@@ -548,10 +561,26 @@ def _find_tags(
         incompatibilities: CandidatesMap,
     ) -> typing.Iterable[tuple[str, Version]]:
         headers = {"accept": "application/vnd.github+json"}
+
+        # Add GitHub authentication if available
+        github_token = os.environ.get("GITHUB_TOKEN")
```

Comment:
The adapter is already doing this, right? Maybe it does make more sense to do it here, where we know we're talking to GitHub?